qoibench.c

```diff
@@ -553,7 +574,10 @@ void benchmark_directory(const char *path, benchmark_result_t *grand_total) {
 
 		if (!has_shown_head) {
 			has_shown_head = 1;
-			printf("## Benchmarking %s/*.png -- %d runs\n\n", path, opt_runs);
+			printf("## Benchmarking %s/*.png -- %d runs ", path, opt_runs);
+			if(opt_forcebpp)
+				printf("-- forcing %d bytes per pixel\n\n", opt_forcebpp);
+			printf("\n\n");
```

Comment:
You may want to adjust the whitespacing for consistent dumps :slightly_smiling_face:
```suggestion
			printf("## Benchmarking %s/*.png -- %d runs", path, opt_runs);
			if(opt_forcebpp)
				printf(" -- forcing %d bytes per pixel", opt_forcebpp);
			printf("\n\n");
```